src/main/java/openfasttrack/importer/tag/TagImporter.java

```diff
@@ -82,15 +86,21 @@ public void runImport()
     private void processLine(final int lineNumber, final String line)
     {
         final Matcher matcher = this.tagPattern.matcher(line);
+        int counter = 0;
         while (matcher.find())
         {
             this.listener.beginSpecificationItem();
-            final SpecificationItemId id = SpecificationItemId.parseId(matcher.group(1));
+            final String generatedName = this.fileName + ":" + lineNumber + "-" + counter;
+            final SpecificationItemId generatedId = SpecificationItemId.createId(matcher.group(1),
+                    generatedName, 0);
+            final SpecificationItemId coveredId = SpecificationItemId.parseId(matcher.group(2));
 
-            LOG.finest(
-                    () -> "File " + this.fileName + ":" + lineNumber + ": found id '" + id + "'");
-            this.listener.setId(id);
+            LOG.finest(() -> "File " + this.fileName + ":" + lineNumber + ": found '" + generatedId
+                    + "' covering id '" + coveredId + "'");
+            this.listener.setId(generatedId);
+            this.listener.addCoveredId(coveredId);
             this.listener.endSpecificationItem();
+            counter++;
```

Comment:
The counter is now only used when calculating the CRC32, no need to make it readable.